### ISSUE TITLE
Can't see through portals from Portal Gun mod

### ISSUE THREAD

**Issue by ChrisLane** [sp614x/optifine#1702]
Forge: 14.23.4.2758
Optifine: 1.12.2_HD_U_E2
Portal Gun: 1.12.2-7.0.2
iChunUtil: 1.12.2-7.1.4

Without Optifine installed, I can see through portals as expected for the portal gun mod.
With Optifine installed, the view through portals is a plain blue colour.
![bugged portals](https://i.imgur.com/fErRtLf.jpg)

**Comment by yourvirtualdad** [comment]
When I first downloaded Optifine, I downloaded the Portal Gun mod right after. Both mods worked great together, and the see through portals were also working great. After a while, I had some fps issues and lag spikes so I decided to fiddle with my Optifine video settings. After switching them around the see through portals weren't working anymore, and the same exact problem is happening to me. I believe the problem is with some of the Video settings, but im not completely sure.
